dcpy/connectors/edm/recipes.py

```diff
@@ -148,6 +148,22 @@ def fetch_dataset(ds: Dataset, local_library_dir=LIBRARY_DEFAULT_PATH) -> Path:
     return target_file_path
 
 
+def read_csv(
+    ds: Dataset, local_cache_dir: Path | None = None, **kwargs
+) -> pd.DataFrame:
+    if ds.file_type != DatasetType.csv:
+        raise Exception(
+            "csv output is currently only format that recipes.read_csv is implemented for"
+        )
```

Comment:
nit: this might be nicer as something like `read_dataset` and only support csv for now

it could then be used in both `recipes.import_dataset` and `dob_cofos.py`